src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImage.groovy

```diff
@@ -108,6 +108,15 @@ class DockerBuildImage extends AbstractDockerRemoteApiTask implements RegistryCr
     @Optional
     final Property<Long> shmSize = project.objects.property(Long)
 
+    /**
+     * With this parameter it is possible to build a special
+     * stage in a multi stage Docker file. This is available with Docker 17.05
+     * @since 4.9.0
```

Comment:
This needs to be 4.10.0. The version 4.9.0 has already been released.

Reply:
changed version